components/Navigation.js

```diff
@@ -92,65 +92,58 @@ export default function Navigation(props) {
       <p>Logo will go here</p>
       <div className={classes.toolbar} />
       <Divider />
-      <List 
-      component="nav"
-      aria-labelledby="nested-list-subheader"
-      className={classes.root}
-    >
+      <List component="nav" aria-labelledby="nested-list-subheader" className={classes.root}>
         {navNodes.map((node) => {
           // console.log(node.label)
-          var isSubMenItem = (node.label === 'Gallery')
-          var subMListItem = (node.label === 'Travel' | node.label === 'Videography')
+          var isSubMenItem = node.label === 'Gallery';
```

Comment:
Just a nit pick: it's always a better idea to use const instead of var/let wherever you can. You don't need to change it, just keep it in mind for the future.